snapshot/snapshotfs/upload.go

```diff
@@ -39,6 +39,9 @@ type Uploader struct {
 	// ignore file read errors
 	IgnoreFileErrors bool
 
+	// ignore directory read errors
+	IgnoreDirErrors bool
```

Comment:
Does it make sense to continue having these fields in the `Uploader`? Can they be easily removed or are there many dependencies? (I have not yet looked).

Reply:
you know what, I added this as the very first shot at a fix, never used it anywhere, and forgot to take it out. I can either add it as another condition for ignoring directories, or remove it.

`IgnoreFileErrors` is set in one place currently, `command_snapshot_migrate.go`.